workspaces/simon-game/src/app/components/Box.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+type Props = { color?: string; height?: number; width?: number };
+
+export function Box(props: Props) {
+  return (
+    <div
+      style={{
+        backgroundColor: props.color ?? "lightblue",
+        height: props.height ?? 100,
+        width: props.width ?? 100,
+        borderRadius: 20, // rounded corners
+        margin: 10, // spaces
```

Comment:
Although this does the job, I'd actually recommend to define the margin as a prop, and not bake any margin into the component by default. Reason being, it's better if the default behavior of a component is "shrink-wrapped". A component should decide what happens inside its boundaries, but leave what happens outside its boundaries to each individual callsite. How much margin looks nice is better expressed in the place where the box is used, not within the box itself.

Reply:
That makes sense to me, I'll keep it in mind going forward. Updated!